independent-projects/arc/processor/src/main/java/io/quarkus/arc/processor/FieldDescriptors.java

```diff
@@ -12,6 +13,22 @@
     static final FieldDescriptor QUALIFIERS_IP_QUALIFIERS = FieldDescriptor.of(Qualifiers.class, "IP_DEFAULT_QUALIFIERS",
             Set.class);
 
+    static final FieldDescriptor ANNOTATION_LITERALS_EMPTY_CLASS_ARRAY = FieldDescriptor.of(AnnotationLiterals.class,
+            "EMPTY_CLASS_ARRAY",
+            Class[].class);
+
+    static final FieldDescriptor ANNOTATION_LITERALS_EMPTY_STRING_ARRAY = FieldDescriptor.of(AnnotationLiterals.class,
+            "EMPTY_STRING_ARRAY",
+            String[].class);
+
+    static final FieldDescriptor ANNOTATION_LITERALS_EMPTY_LONG_ARRAY = FieldDescriptor.of(AnnotationLiterals.class,
+            "EMPTY_LONG_ARRAY",
+            String[].class);
```

Comment:
Is it normal it's String here?

Reply:
Eh, of course not. I was just too tired...